app/src/main/res/layout/hero_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
```

Comment:
match_parent en el width asi pon un simple padding quedaria centrado y sin desorientacion
En caso de que en vez de 5 columnas tuviesemos 2 se verian los items orientados a la Izquierda.

Reply:
@JorgeGreek with wrap_content it only takes the necessary space to show the item